sdk/devcenter/azure-developer-devcenter/tests/testcase_async.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from devtools_testutils import AzureTestCase
+from azure.developer.devcenter.aio import DevCenterClient
+
+
+class DevcenterAsyncTest(AzureTestCase):
+    def create_client(self, tenant_id, dev_center):
+        credential = self.get_credential(DevCenterClient)
+        return DevCenterClient(
```

Comment:
same with https://github.com/Azure/azure-sdk-for-python/pull/26696/files#r997652084

Reply:
See above.